[WKCI] Bring up OS 26 builders and testers.

bls1999 opened the pull request:
#### 740baa61176069ad445ddd2bb6048f955c987226
<pre>
[WKCI] Bring up OS 26 builders and testers.
<a href="https://bugs.webkit.org/show_bug.cgi?id=299503">https://bugs.webkit.org/show_bug.cgi?id=299503</a>
<a href="https://rdar.apple.com/161292280">rdar://161292280</a>

Reviewed by Ryan Haddad.

Bringing up the CI infrastructure to support building and testing on OS 26.

* Tools/CISupport/build-webkit-org/config.json: Add Tahoe queues, update relevant Sequoia and embedded queues.
* Tools/CISupport/build-webkit-org/factories_unittest.py: Ditto.
* Tools/CISupport/build-webkit-org/wkbuild.py: Ditto.
* Tools/CISupport/build-webkit-org/wkbuild_unittest.py: Ditto.
* Tools/CISupport/ews-build/config.json: Ditto.
* Tools/CISupport/ews-build/factories_unittest.py: Ditto.

Changing the dashboard to reflect these changes:

* Tools/CISupport/build-webkit-org/public_html/dashboard/Scripts/Dashboard.js
* Tools/CISupport/build-webkit-org/public_html/dashboard/Scripts/WebKitBuildbot.js
* Tools/CISupport/build-webkit-org/public_html/dashboard/Styles/Main.css

Canonical link: <a href="https://commits.webkit.org/302052@main">https://commits.webkit.org/302052@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/7120c6b75fadbb8f9b8d1e10da975ada261cd75e

| Misc | iOS, visionOS, tvOS & watchOS  | macOS  | Linux |  Windows | Apple Internal |
| ----- | ---------------------- | ------- |  ----- |  --------- | ------ |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/126932 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/131/builds/46569 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/138/builds/37557 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/133936 "Built successfully") | [✅ 🛠 win](https://ews-build.webkit.org/#/builders/59/builds/78512 "Built successfully") | [✅ 🛠 ios-apple](https://ews-bridge.webkit.apple.com/builds/sw/T-276/f2c6d2ed-bd8b-4d70-a499-64172ae81bd0/eb215b6d-80db-47c1-b463-c2b4d5a1c13d) 
| [✅ 🧪 bindings](https://ews-build.webkit.org/#/builders/9/builds/128803 "Passed tests") | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/130/builds/47189 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/123/builds/55100 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/96591 "Passed tests") | [❌ 🧪 win-tests](https://ews-build.webkit.org/#/builders/60/builds/65095 "") | [✅ 🛠 mac-apple](https://ews-bridge.webkit.apple.com/builds/sw/T-276/f2c6d2ed-bd8b-4d70-a499-64172ae81bd0/4740d1a0-0201-4a56-8947-f701c28a1917) 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/129880 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/132/builds/37763 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/113531 "Passed tests") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/77105 "Passed tests") | | [✅ 🛠 vision-apple](https://ews-bridge.webkit.apple.com/builds/sw/T-276/f2c6d2ed-bd8b-4d70-a499-64172ae81bd0/260c556b-cc1c-479a-b5e4-c714632c019d) 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/133/builds/36625 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/135/builds/31700 "Passed tests") | [✅ 🛠 wpe-cairo](https://ews-build.webkit.org/#/builders/65/builds/77330 "Built successfully") | | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/107599 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/136/builds/32017 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/136461 "Built successfully") | | 
| | [✅ 🛠 vision](https://ews-build.webkit.org/#/builders/128/builds/53592 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/122/builds/41261 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/105107 "Passed tests") | | 
| [✅ 🧪 services](https://ews-build.webkit.org/#/builders/28/builds/127061 "Passed tests") | [✅ 🛠 vision-sim](https://ews-build.webkit.org/#/builders/121/builds/54095 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/109894 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/104799 "Passed tests") | | 
| | [✅ 🧪 vision-wk2](https://ews-build.webkit.org/#/builders/126/builds/50314 "Passed tests") | [✅ 🧪 mac-intel-wk2](https://ews-build.webkit.org/#/builders/137/builds/28648 "Passed tests") | [✅ 🛠 playstation](https://ews-build.webkit.org/#/builders/134/builds/51072 "Built successfully") | | 
| [✅ 🛠 🧪 unsafe-merge](https://ews-build.webkit.org/#/builders/22/builds/19985 "Built successfully and passed tests") | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/127/builds/53524 "Built successfully") | | | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/125/builds/52765 "Built successfully") | | | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/129/builds/56099 "Built successfully") | | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/124/builds/54524 "Built successfully") | | | | 
<!--EWS-Status-Bubble-End-->

ryanhaddad commented on Tools/CISupport/build-webkit-org/config.json:
```diff
@@ -24,73 +24,73 @@
-                  { "name": "bot179", "platform": "mac-sequoia" },
-                  { "name": "bot184", "platform": "mac-sequoia" },
+                  { "name": "bot165", "platform": "mac-tahoe" },
+                  { "name": "bot178", "platform": "mac-tahoe", "notes": "Macmini8,1 on Sequoia building for Tahoe" },
```
What do these notes mean? I don't see this hostname assigned to any builder.

bls1999 replied:
We intend to use Macmini8,1s for building on Tahoe. At the moment, we're encountering a clang crash that I'm investigating, which is blocking our use of these machines for this purpose. I've kept the notes because we still intend to use them once we're unblocked.

bls1999 committed:
[WKCI] Bring up OS 26 builders and testers.
https://bugs.webkit.org/show_bug.cgi?id=299503
rdar://161292280

Reviewed by Ryan Haddad.

Bringing up the CI infrastructure to support building and testing on OS 26.

* Tools/CISupport/build-webkit-org/config.json: Add Tahoe queues, update relevant Sequoia and embedded queues.
* Tools/CISupport/build-webkit-org/factories_unittest.py: Ditto.
* Tools/CISupport/build-webkit-org/wkbuild.py: Ditto.
* Tools/CISupport/build-webkit-org/wkbuild_unittest.py: Ditto.
* Tools/CISupport/ews-build/config.json: Ditto.
* Tools/CISupport/ews-build/factories_unittest.py: Ditto.

Changing the dashboard to reflect these changes:

* Tools/CISupport/build-webkit-org/public_html/dashboard/Scripts/Dashboard.js
* Tools/CISupport/build-webkit-org/public_html/dashboard/Scripts/WebKitBuildbot.js
* Tools/CISupport/build-webkit-org/public_html/dashboard/Styles/Main.css

Canonical link: https://commits.webkit.org/302052@main
